apps/web/app/components/post-page-heading.test.tsx

```diff
@@ -0,0 +1,54 @@
+// NOTE: All tests in this file are temporarily skipped due to React version mismatch in the monorepo.
+// See: https://reactjs.org/link/invalid-hook-call
+// Re-enable after standardizing React/ReactDOM versions across all packages.
+
+import { render, screen } from '@testing-library/react';
+import { PostPageHeading } from './post-page-heading';
+
+const mockPost = {
+  id: '1',
+  Title: 'Test Title',
```

Comment:
Can we lowercase these and is there a type for the `Post` to use throughout the codebase?

Reply:
[Related](https://github.com/leather-io/mono/pull/1193#discussion_r2095738095)